### DISCUSSION TITLE
Ldap Sync Automation on deployment

### DISCUSSION THREAD

**Discussion by compgeniuses** [DefGuard/defguard#690] [Q&A] [answered]
taking a look at the Ldap sync options provied here https://defguard.gitbook.io/defguard/admin-and-features/ldap-synchronization-setup/settings-table

I was wondering, is it possible to pre-configure the Ldap sync parameters via environment variables during Docker Based installation?
If so, What exactly should be used as the Env variables.

Equally, is this also possible via a config File?

**Comment by kchudy** [comment, marked as answer]
All LDAP-related config options are in the database. You cannot configure it using env variables or config files.

**Reply by compgeniuses** [reply]
Would be great, if they were, would simplify Automation. However, are there existing documentation, on the same, of how to inject, the DB to get them working

**Reply by kchudy** [reply]
There's no documentation on the database entries for LDAP configuration, but they're pretty self-explanatory. Please inspect the _settings_ table in your Core's database. Look for rows with _ldap__ prefix as keys.

**Comment by compgeniuses** [comment]
Ok, Since you mentioned V2 is around the conner, within a month, with many changes, I would wait for the release, then attempt Automated Implementation of Defguard for Nethserver 8 as Announced here: https://community.nethserver.org/t/defguard-2fa-mfa-wireguard-vpn-with-sso/24144

We would be happy to take any support Offered, if it will enable us to Have a proper Implementation of the Project, for companies, and end users, looking for a straightforward way to selfHost.
